plugins/Actions/Archiver.php

```diff
@@ -165,6 +166,25 @@ protected function archiveDaySiteSearchActions($rankingQueryLimit)
         }
     }
 
+    protected function archiveDaySearchCategoryActions()
+    {
+        $where = "%s.search_cat != ''";
```

Comment:
should we also check if `search_cat IS NOT NULL`?

Reply:
makes sense as the column is nullable